cmd/fakemachine/main.go

```diff
@@ -18,6 +19,7 @@ type Options struct {
 	CPUs        int               `short:"c" long:"cpus" description:"Number of CPUs for the fakemachine"`
 	ScratchSize string            `short:"s" long:"scratchsize" description:"On-disk scratch space size (with a unit suffix, e.g. 4G); if unset, memory backed scratch space is used"`
 	ShowBoot    bool              `long:"show-boot" description:"Show boot/console messages from the fakemachine"`
+	KernelPath  string            `short:"k" long:"kernel" description:"kernel to use"`
```

Comment:
as stated in my review of https://github.com/go-debos/debos/pull/376 it'd be much better to implement this as a `backed-option`, so something like:

`--backend-option kernel-path:kernel.img`

```
if err = m.SetBackendOption("key", "value"); err != nil {
    // show error to user
}
```